src/compas/geometry/bestfit/bestfit_numpy.py

```diff
@@ -122,8 +122,11 @@ def f(c):
         Ri = dist(*c)
         return Ri - Ri.mean()
 
-    xm = mean(x)
-    ym = mean(y)
+    # The addition of epsilon is to promote convergence within leastsq,
+    # which seems to strongly dislike (0, 0) as an initial guess.
+    epsilon = .00001
+    xm = mean(x) + epsilon
+    ym = mean(y) + epsilon
```

Comment:
this makes more sense to me than the previous version, but what if `mean(x) == -epsilon` and `mean(y) == -epsilon` :)

Reply:
yeah... so `mean(x)` empirically is on the order of e-15. i'm not sure that i could prove that is always the case, but the definition of `x` sure makes it seem like `mean(x)` will always be quite tiny. in my experiments, an epsilon of `0.0000001` would reveal the instability, while `0.000001` didn't.  so i went with `.00001`. i could just go with `xm = ym = epsilon` or even `xm = ym =  1`.  1 takes a smidge longer to converge, but it would work.